pkg/k8s/deployments/crud.go

```diff
@@ -142,6 +144,36 @@ func Deploy(d *appsv1.Deployment, forceCreate bool, client *kubernetes.Clientset
 	return nil
 }
 
+//UpdateOktetoRevision updates the okteto version annotation
+func UpdateOktetoRevision(ctx context.Context, d *appsv1.Deployment, client *kubernetes.Clientset) error {
+	tries := 0
+	ticker := time.NewTicker(200 * time.Millisecond)
+	for tries < maxRetriesUpdateRevision {
```

Comment:
what is the loop for?

Reply:
we need to get the revision number that k8s generates on each deployment update, but it usually takes a few ms to generate it